docs/README.es.md

```diff
@@ -31,6 +31,57 @@
 
 <br></br>
 
+##### Traducciones
```

Comment:
@JoseNoriegaa we actually don't want the translations table on other pages sense it is too much to maintain. Please remove, if you will (:

Reply:
I removed the translation table, the PR is up to date.